funcx_sdk/funcx/utils/errors.py

```diff
@@ -96,16 +96,6 @@ def __repr__(self):
         return f"HTTP request failed: {self.message}"
 
 
-class InvalidScopeException(FuncxError):
```

Comment:
This is used in `funcx_sdk/funcx/sdk/search.py`. We should double check this.

Reply:
This definition is a duplicate of `InvalidScopeException` defined 10 lines up, in lines 79-86.
I don't see any distinction between the two.

Reply:
Silly that we have such a duplication :)